.kokoro/upload-docs.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+# build/test docs (Node 8.16.0 is currently installed on Python image).
+export NPM_CONFIG_PREFIX=/home/node/.npm-global
+npm install
+npm run docs-test
+
+# create docs.metadata, based on package.json and .repo-metadata.json.
+npm i json@9.0.6 -g
+python3 -m pip install gcp-docuploader
+python3 -m docuploader create-metadata \
```

Comment:
So - I'm ok with this, but - I wonder if we couldn't template this out somehow? I know there are other places we tend to inject the metadata.

Reply:
my preference would be pointing `docuploader` at `.repo-metadata.json`, rather than copying the values over; perhaps we could keep a `version` field up-to-date in `.repo-metadata.json`; I think this is the only missing field right @busunkim96?

Reply:
Huh - I do worry a little about replicating too much data here that's already available in `package.json`, but I hear ya.

Reply:
I'm just thinking it would be annoying for `gcp-uploader` to know about the different meta information files; It's nice that `.repo-metadata.json` consolidates.

Reply:
I'd rather it be annoying 7 times in gcp-uploader rather than annoying 147 times 👅